### ISSUE TITLE
Waypoints getting dragged around while screen scrolling on mobile

### ISSUE THREAD

**Issue by Bertie690** [sirgwain/craig-stars#424]
Real annoying and IDK why it happens, but it usually makes my waypoints turn into a tangled knotted mess.

**Comment by sirgwain** [comment]
yeah, and no undo button. not sure how to fix it. the input to drag a waypoint vs the input to pan is the same.
